common/changes/@microsoft/api-extractor/patch-1_2023-09-20-22-22.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Don't strip out @alpha items when generating API reports.",
```

Comment:
@dmichon-msft for future reference, this should have been a MINOR version bump.  It is a significant behavioral change that may cause previously undisclosed content to appear on a public website.

Reply:
Gah, I've gotten so used to the Rush projects where the version bump isn't part of the PR that I forgot to check that.

Reply:
What is the process for correcting this? I'm happy to put up a PR fixing this if I can get guidance.